test/integration/karma.conf.js

```diff
@@ -0,0 +1,11 @@
+const baseConfig = require('../../karma.conf');
+
+module.exports = function(config) {
+  baseConfig(config);
+  config.set({
+    files: [
+      './**_test.js',
```

Comment:
You may want to explicitly bring in only the integration test as future tests will have goog.module() syntax which may cause issues for Karma running this config.

Reply:
Actually, this does only bring in the integration tests. The file paths are relative, and this file is in the 'test/integration' directory, so it should only grab test files placed in the integration directory.